crates/ruff_linter/src/rules/flake8_pyi/rules/redundant_literal_union.rs

```diff
@@ -59,7 +59,7 @@ impl Violation for RedundantLiteralUnion {
 
 /// PYI051
 pub(crate) fn redundant_literal_union<'a>(checker: &mut Checker, union: &'a Expr) {
-    let mut literal_exprs = Vec::new();
+    let mut typing_literal_exprs = Vec::new();
```

Comment:
The old name was confusing. The vector refers to all the elements inside a `typing.Literal`. Maybe `typing_literal_values`?